projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -60,88 +60,6 @@ function register_block() {
 					),
 					'align'   => array( 'wide', 'full' ),
 				),
-				'attributes'      => array(
```

Comment:
Attributes were added back to server-side registration as a temporary workaround (https://github.com/Automattic/jetpack/pull/36695).